x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/optimizer/rules/SubstituteSurrogates.java

```diff
@@ -133,28 +133,4 @@ protected LogicalPlan rule(Aggregate aggregate) {
 
         return plan;
     }
-
-    public static String temporaryName(Expression inner, Expression outer, int suffix) {
```

Comment:
Moving this to `LogicalPlanOptimizer` since it's used more widely than just here.